crates/service/src/index/indexing/mod.rs

```diff
@@ -130,11 +161,11 @@ impl<S: G> DynamicIndexing<S> {
         }
     }
 
-    pub fn vector(&self, i: u32) -> &[S::Scalar] {
+    pub fn content(&self, i: u32) -> S::VectorRef<'_> {
```

Comment:
Let's keep the name of `vector` and `open`. Afterall we already use the two names everywhere.

Reply:
done in new commit